lib/docs/scrapers/mkdocs.rb

```diff
@@ -0,0 +1,5 @@
+module Docs
+  class Mkdocs < Scraper
+    self.abstract = true
```

Comment:
Things that should be common among all Mkdocs scrapers should be here:

<details><summary>Suggested change</summary>

```suggestion
    self.abstract = true
    self.type = 'mkdocs'
    html_filters.push 'mkdocs/clean_html'

    private

    def handle_response(response)
      # Some scrapped urls don't have ending slash
      # which leads to page duplication
      if !response.url.path.ends_with?('/') && !response.url.path.ends_with?('index.html')
        response.url.path << '/'
      end
      super
    end
```
</details>

Reply:
Should `RestFramework` be subclassed from `Mkdocs`?